src/utils/config.nix

```diff
@@ -7,7 +7,9 @@ let
   #   - attrset (no changes)
   loadAttrs = input:
     if b.isPath input
-    then b.fromJSON (b.readFile input)
+    # discarding context here should be fine since we read the text from
+    # a path, which will be realized and nothing else will need to be realized
+    then b.fromJSON (b.unsafeDiscardStringContext (b.readFile input))
```

Comment:
I did this because `fromJSON` would complain about the string having a reference to a nix store path. This specifically happens with running the example tests. I know that the nix store path might not exist and it might just have been forged, but I don't think there should be an issue with this in tests. I don't really like this being here, so I'd rather remove it but otherwise the tests don't work.